README.md

```diff
@@ -42,7 +42,7 @@ just include `dist/js/vue-dynamic-form.min.js` on the page, after Vue:
 ```
 
 In this case the component will be automatically registered. 
-See [examples/index/html](examples/index.html).
+See [docs/index.html](docs/index.html).
```

Comment:
I put this in the `docs` folder so that we can configure GH pages to serve this example (you can configure /docs as the GH pages source, but not arbitrarily named folders unfortunately).